src/Features/CSharp/Portable/MakeLocalFunctionStatic/GetCaptures.cs

```diff
@@ -0,0 +1,189 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeGeneration;
+using Microsoft.CodeAnalysis.CSharp.CodeGeneration;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.FindSymbols;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+
+
+
+
+namespace Microsoft.CodeAnalysis.CSharp.GetCapturedVariables
+{
+    [Shared]
+    [ExportLanguageService(typeof(GetCaptures), LanguageNames.CSharp)]
+    internal class GetCaptures : ILanguageService
+    {
+        private static readonly char[] s_underscore = { '_' };
+        private static readonly SyntaxGenerator s_generator = CSharpSyntaxGenerator.Instance;
+
+
+
+        internal async Task<Solution> CreateParameterSymbolAsync(Document document, LocalFunctionStatementSyntax localfunction, CancellationToken cancellationToken)
```

Comment:
Why do we need to return a new solution here? Return the new Document should suffice, right?